.github/workflows/master.yml

```
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Canonical Ltd.
+
+name: Master workflow
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build-dbtestapp:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Build
+        run: |
+          cd mongoapi/dbtestapp/
+          go build
```

Should not this be as follows?
```suggestion
          go build ./...
```

---

Done